libs/vre/shared/app-representations/src/lib/third-party-iiif/third-party-iiif.ts

```diff
@@ -57,15 +57,15 @@ export class IIIFUrl {
       return false;
     }
 
-    if (!this.rotation_ex.test(this._rotationSegment)) {
+    if (!this.rotation_ex.test(this._rotationSegment) && this.rotation_ex.test(decodeURI(this._rotationSegment))) {
```

Comment:
```suggestion
    if (!this.rotation_ex.test(this._rotationSegment) && !this.rotation_ex.test(decodeURIComponent(this._rotationSegment))) {
```
Although it works you should use `decodeURIComponent` instead and negate the test result, because in fact it decodes the segment correctly. Here is the documentation for encoding function which explain the scope: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
Please update below statements too.